specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotRecommendations.json

```diff
@@ -0,0 +1,237 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/iotSecuritySolutions/{solutionName}/iotRecommendations/{iotRecommendationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get IoT Recommendation": {
+            "$ref": "./examples/IotRecommendations/GetIoTRecommendation.json"
+          }
+        },
+        "tags": [
+          "IoT Security Recommendations"
+        ],
+        "operationId": "IotRecommendations_Get",
+        "description": "Get IoT recommendation",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SolutionName"
+          },
+          {
+            "in": "path",
+            "name": "iotRecommendationId",
+            "required": true,
+            "type": "string",
+            "description": "Id of the recommendation"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "IoT recommendation",
+            "schema": {
+              "$ref": "#/definitions/IotRecommendation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IotRecommendationList": {
+      "description": "List of IoT recommendations",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotRecommendation"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "When available, follow the URI to get the next page of data"
+        },
+        "totalCount": {
+          "readOnly": true,
+          "format": "int32",
+          "type": "integer",
+          "description": "Total count of recommendations that conforms with the given filter options (not affected by page size)"
+        }
+      }
+    },
+    "IotRecommendation": {
+      "type": "object",
+      "description": "IoT recommendation",
+      "properties": {
+        "properties": {
```

Comment:
this is missing the top level resource properties "type", "name", and "id".  Please inherit from the common "Resource" definition like iotRecommendationTypes does